docs/reference/data-streams/use-a-data-stream.asciidoc

```diff
@@ -426,12 +431,98 @@ POST /_reindex
 [[update-delete-docs-in-a-data-stream]]
 === Update or delete documents in a data stream
 
-Data streams are designed to be <<data-streams-append-only,append-only>>. This
-means you cannot send update or deletion requests for existing documents to a
-data stream. However, you can send update or deletion requests to the backing
-index containing the document.
+You can use the following requests to update or delete documents in a data
+stream:
+
+* <<docs-update-by-query,Update by query API>> requests containing a query
```

Comment:
Minor: I thought the "Update by query API requests containing..." sequence read a little oddly.